shared/constants/platform-notifications.ts

```diff
@@ -0,0 +1,149 @@
+import { CHAIN_IDS } from './network';
+
+export enum TRIGGER_TYPES {
+  FEATURES_ANNOUNCEMENT = 'features_announcement',
+  METAMASK_SWAP_COMPLETED = 'metamask_swap_completed',
+  ERC20_SENT = 'erc20_sent',
+  ERC20_RECEIVED = 'erc20_received',
+  ETH_SENT = 'eth_sent',
+  ETH_RECEIVED = 'eth_received',
+  ROCKETPOOL_STAKE_COMPLETED = 'rocketpool_stake_completed',
+  ROCKETPOOL_UNSTAKE_COMPLETED = 'rocketpool_unstake_completed',
+  LIDO_STAKE_COMPLETED = 'lido_stake_completed',
+  LIDO_WITHDRAWAL_REQUESTED = 'lido_withdrawal_requested',
+  LIDO_WITHDRAWAL_COMPLETED = 'lido_withdrawal_completed',
+  LIDO_STAKE_READY_TO_BE_WITHDRAWN = 'lido_stake_ready_to_be_withdrawn',
+  ERC721_SENT = 'erc721_sent',
+  ERC721_RECEIVED = 'erc721_received',
+  ERC1155_SENT = 'erc1155_sent',
+  ERC1155_RECEIVED = 'erc1155_received',
+}
+
+const chains = {
```

Comment:
Nit: because we refer to `ETHEREUM` as `MAINNET` throughout the `shared/constants/network` file, and that is used throughout the codebase, it might be best to name this `chains.ETHEREUM` property as `chains.MAINNET`